metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -237,7 +237,7 @@ object UserConfiguration {
       ),
       UserConfigurationOption(
         "enable-semantic-highlighting",
-        "false",
+        "true",
         "false",
         "Use semantic tokens highlight (experimental).",
```

Comment:
I guess we can stop calling it `experimental` if it's the default.

Reply:
Fixed! good catch!